### PR TITLE
Update readme with more correct information

### PR DESCRIPTION
**Pull request by vmpstr**
Added a disclaimer section to the readme pointing to issues and cheatsheet. Also updated various wording to reduce confusion. PTAL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,40 +2,120 @@
...
-These techniques can work in many cases but have some drawbacks and limitations. These include:
-a. [\[3\]](#foot-notes) and [\[4\]](#foot-notes) usually means that such content is not available to user-agent features. Also, content that is merely placed offscreen may or may not have rendering cost (it depends on browser heuristics), which makes the technique unreliable.
-b. Caching intermediate rendering state is [hard work](https://martinfowler.com/bliki/TwoHardThings.html), and often has performance limitations and cliffs that are not obvious to developers. Similarly, relying on the browser to avoid rendering for content that is clipped out or not visible is sometimes not reliable, as it's hard for the browser to efficiently detect what content is actually visible.
+* Fast display of large HTML documents (examples: HTML one-page spec, other long
+  documents)
+* Deep links and searchability into pages with hidden content (example: mobile
+  Wikipedia)
+* Scrollers with a large amount of content, without resorting to virtualization
+  (examples: twitter feed, codemirror documents)
+* Pre-loading and rendering of non-visible content asynchronously to prepare it
```
Comment (chrishtr) [inline README.md]: How about re-phrasing this bullet as:

* Single-page app transitions

File: README.md [hunk 2]
```diff
@@ -2,40 +2,120 @@
...
+pair that with a guarantee that when later rendered, it would not invalidate
+more than a small amount of style, layout or paint in the rest of the document.
+
+## Disclaimer
+
+As the proposed features evolve, several competing API shapes might be
+considered at the same time, the decisions on particular behaviors might not be
+finalized, and some documentation may be out of date.
+
+For the latest agreed-upon behavior and API state, please consult the
```
Comment (chrishtr) [inline README.md]:
```suggestion
For the latest implemented behavior and API state, please consult the
```

File: README.md [hunk 3]
```diff
@@ -2,40 +2,120 @@
...
+[cheatsheet](https://github.com/WICG/display-locking/blob/master/cheatsheet.md).
+
+For behaviors being discussed, as well as questions and other discussions,
+please look over the [issues](https://github.com/WICG/display-locking/issues)
+
+The rest of this document talks about one particular set of proposed features.
```
Comment (chrishtr) [inline README.md]:
```suggestion
The rest of this document talks about one particular implementation option.
```

File: README.md (hunk 1 above)
Comment (vmpstr) [reply README.md]: Done.

**Comment by vmpstr** [comment]
PTAL

**Pull request merged**